ilc/client/ClientRouter.js

```diff
@@ -150,7 +159,8 @@ export default class ClientRouter {
         const {specialRole} = this.#router.match(pathname);
 
         if (specialRole === null) {
-            this.#singleSpa.navigateToUrl(href);
+            const processedUrl = this.#urlProcessor.process(href);
```

Comment:
You don't need it here, see https://github.com/single-spa/single-spa/blob/d89173b6cadd4ba2e581a648a5ee0821488a71d8/src/navigation/navigation-events.js#L18

Reply:
It does not work without it unfortunately

Reply:
It was a mistake in the code
It works
I removed it from here